.buildkite/scripts/make_image.sh

```diff
@@ -33,12 +33,25 @@ case "$BUILD_TYPE" in
         DOCKER_NAME="${BASE_DOCKER_NAME}-${IMAGE_ARCH}_testnet_${VERSION}"
         LATEST_TAG="latest-val-${IMAGE_ARCH}"
         ;;
+    "val_tpm")
+        echo "Doing a testnet validator with TPM image build for ${IMAGE_ARCH}"
+        DOCKER_BUILD_ARGS="--build-arg BUILDER_IMAGE=$BASE_IMAGE --build-arg RUNNER_IMAGE=$BASE_IMAGE --build-arg REBAR_BUILD_TARGET=docker_testval_tpm $DOCKER_BUILD_ARGS"
+        BASE_DOCKER_NAME="validator_tpm"
```

Comment:
This is where you want to define your extra runner and build packages - should be for each tag or some other mechanism that makes sense.